packages/origin-247-transfer/src/repositories/EnergyTransferRequest.entity.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    CreateDateColumn,
+    Entity,
+    PrimaryGeneratedColumn,
+    UpdateDateColumn,
+    Column
+} from 'typeorm';
+import { EnergyTransferRequestAttrs, TransferValidationStatus } from '../EnergyTransferRequest';
+
+export const tableName = 'energy_transfer_request';
+
+@Entity(tableName)
+export class EnergyTransferRequestEntity implements EnergyTransferRequestAttrs {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @CreateDateColumn()
+    createdAt: Date;
+
+    @UpdateDateColumn()
+    updatedAt: Date;
```

Comment:
Improvement suggestion, which has a bit different custom implementation:


https://github.com/energywebfoundation/origin/blob/121fafd84515d482b42b477fe9f8c617481d22b2/packages/utils/origin-backend-utils/src/entity/ExtendedBaseEntity.ts#L4-L24

```js
import { ExtendedBaseEntity } from '@energyweb/origin-backend-utils';

export class EnergyTransferRequestEntity extends ExtendedBaseEntity ....
```

Reply:
What is the gain of the custom transformers?

I'm not using `ExtendedBaseEntity` on purpose, because it uses `BaseEntity` and this is Active Records. I don't like Active Records. But I can add timestamptz field or custom transformers, just I would like to know what purpose they serve

Reply:
Alright, makes sense. The only benefit here would be storing tz info, but if you would need that you can just set the type to timestamptz.

Reply:
Oh right, that makes sense, I will change it. I haven't run SBE migrations yet, so we are safe to make breaking change

Reply:
fixed